scripts/build/write-js.ts

```diff
@@ -1,7 +1,7 @@
 import rollupTypescript from "@rollup/plugin-typescript";
 import { promises as fs } from "fs";
 import { rollup } from "rollup";
-import packageJson from "../../package.json" assert { type: "json" };
+import packageJson from "../../package.json" with { type: "json" };
```

Comment:
This changed in [typescript 5.3](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-3.html) but was legacy